### PR TITLE
NeZha-D1s/Ubuntu: update test report for Ubuntu 24.10 and Ubuntu 24.04.1 LTS

### PR DESCRIPTION
**Pull request by stydxm**

### CODE REVIEW

File: NeZha-D1s/README.md [hunk 1]
```diff
@@ -16,22 +16,26 @@ cpu_core: XuanTie C906
...
-| Software Category     | Package Name | Test Results (Test Report) |
-| --------------------- | ------------ | -------------------------- |
-| Tina-Linux Image Boot | N/A          | [CFT][Tina]                |
-| RT-Thread Image Boot  | N/A          | [CFT][RT-Thread]           |
-| Ubuntu Image Boot     | N/A          | [Success][Ubuntu]          |
+| Software Category     | Package Name | Test Results (Test Report)   |
+| --------------------- | ------------ | ---------------------------- |
+| Tina-Linux Image Boot | N/A          | [CFT][Tina]                  |
+| RT-Thread Image Boot  | N/A          | [CFT][RT-Thread]             |
+| Ubuntu Image Boot     | N/A          | [Success][Ubuntu][Ubuntu LTS]|
```
Comment (KevinMX) [inline NeZha-D1s/README.md:36]: You're sure to write it like this?
`[aaa][bbb]` and then `[bbb]: url` at the bottom means to reference the link in `bbb` and use it for `aaa`, while only displaying `aaa`.

**Review by KevinMX — CHANGES_REQUESTED** [review]

**Comment by KevinMX** [comment]
https://github.com/ruyisdk/support-matrix/pull/52/files

These ones should be changed as well.

File: NeZha-D1s/README.md (hunk 1 above)
Comment (stydxm) [reply NeZha-D1s/README.md:36]: sorry, my mistakes

Comment (stydxm) [reply NeZha-D1s/README.md:36]: sorry, my mistakes

**Comment by stydxm** [comment]
> https://github.com/ruyisdk/support-matrix/pull/52/files
> 
> These ones should be changed as well. Missed those in that PR.

I'll create another PR to fix this

**Pull request merged**